### PR TITLE
[DTM] Role based access to bulk import

### PR DESCRIPTION
**Pull request by lra-sag**
Document role based access to bulk import

### CODE REVIEW

**Review by pavanshivajirao — APPROVED** [review]

File: content/dtm/asset-hierarchy-bundle/creating-assets-bulk-import.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ Once the permissions are assigned, the bulk import feature can be used.
 
 {{< c8y-admon-info>}}
 If the permissions are not assigned, the CSV template will not be downloaded correctly.
+
+To initiate bulk import from **Assets** page, you must have either CREATE or ADMIN permission for permission type "Inventory".
```
Comment (MWindel) [inline content/dtm/asset-hierarchy-bundle/creating-assets-bulk-import.md:38]:
```suggestion
To initiate a bulk import from the **Assets** page, you must have either have CREATE or ADMIN permission for the permission type "Inventory". 
```

Comment (lra-sag) [inline content/dtm/asset-hierarchy-bundle/creating-assets-bulk-import.md:38]:
```suggestion
To initiate a bulk import from the **Assets** page, you must either have CREATE or ADMIN permission for the permission type "Inventory".
```

Comment (lra-sag) [reply content/dtm/asset-hierarchy-bundle/creating-assets-bulk-import.md:38]: @MWindel, just removed the extra have before either in the sentence and created another suggestion to commit.

**Review by MWindel — APPROVED** [review]

**Comment by rssag** [comment]
what are the target versions? @lra-sag

**Comment by lra-sag** [comment]
> what are the target versions? @lra-sag
@rssag, 10.19 and above are the target versions at the moment

**Pull request merged**